### ISSUE TITLE
Possible issue with 'Get AWS access' section on 'Get started on GOV.UK' page

### ISSUE THREAD

**Issue by jon-kirwan** [alphagov/govuk-developer-docs#3197]
Hello, I have come across a possible issue with the _Get started on GOV.UK_ page.

On step 7, (_Get access to integration infrastructure_) https://github.com/alphagov/govuk-developer-docs/blob/6f9b0bd93f5b6269289b73d804a656b43e0a8448/source/manual/get-started.html.md#get-access-to-integration-infrastructure, it's mentioned:

- To get access, you must add your email address to the list of `role_user_user_arns` users in the `govuk-aws-data` GitHub repo and
- Use the same `<account-ID>` as other entries in the `role_user_user_arns` section. Use the same `<account-ID>` as other entries in the `role_user_user_arns` section.

I found that the policy for this role did not provide me with permissions to complete the rest of the steps in _Access AWS for the first time_. Intead I needed to add my email address to the section `role_poweruser_user_arns`.

Otherwise, I would come across the following error when I'd run `gds aws govuk-integration-poweruser -l`:

```
aws-vault: error: login: Failed to get credentials for govuk-integration-poweruser: 
AccessDenied: User: arn:aws:iam::622626885786:user/jon.kirwan@digital.cabinet-office.gov.uk 
is not authorized to perform: sts:AssumeRole on resource: arn:aws:iam::210287912431:role/jon.kirwan-poweruser 
status code: 403, request id: 8436e068-5a27-4ba9-a460-bd4c6f6acb21
```
Thanks 👍

**Comment by ollietreend** [comment]
@jon-kirwan it looks like we both spotted the same issue with the 'Get started' docs.

I resolved it with the PR #3318, so I'm going to close this issue.

**Comment by jon-kirwan** [comment]
> @jon-kirwan it looks like we both spotted the same issue with the 'Get started' docs.
> 
> I resolved it with the PR #3318, so I'm going to close this issue.

Thanks @ollietreend - good to know!
